test/cookie.test.js

```diff
@@ -244,6 +275,34 @@ test('cookies signature', (t) => {
       t.deepEqual(JSON.parse(res.body), { unsigned: 'foo' })
     })
   })
+
+  t.test('unsignCookie decorator with custom signer', t => {
```

Comment:
```suggestion
  test('unsignCookie decorator with custom signer', t => {
```

Reply:
Ok, moved this one out

Reply:
I didn't see an issue. They were subtests covering an overall parent subject.